cmake/projects/chaiscript/hunter.cmake

```diff
@@ -0,0 +1,30 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+  PACKAGE_NAME
+  chaiscript
+  VERSION
+  "6.0.0"
```

Comment:
Make sure that version matches real git tag, I expect something like `6.0.0-pN` here.

Reply:
The real tag is v6.0.0-hunter

Reply:
> I will look at some other projects, ZMQPP uses 4.1.2 in hunter.cmake, for example... and has a branch 4.1.2-cmake

It's probably issue in ZMQPP, do not rely on it.

> So v6.0.0 should be used in this example?

No, if you have branch `hunter-6.0.0` and created tag `v6.0.0-p0` use version `6.0.0-p0`.